samples/client/petstore/python-asyncio/petstore_api/rest.py

```diff
@@ -85,8 +85,9 @@ def __init__(self, configuration, pools_size=4, maxsize=None):
                 connector=connector
             )
 
-    def __del__(self):
-        asyncio.ensure_future(self.pool_manager.close())
+    def close(self):
+        if not self.pool_manager.closed:
+            asyncio.ensure_future(self.pool_manager.close())
```

Comment:
Can you add a test that checks the pool manager and ensures that it is closed like we did here:
samples/client/petstore/python-experimental/tests/test_api_client.py?

Reply:
There is such test. Sorry, I forgot to commit it before.

Reply:
Test is below